src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -52,15 +58,55 @@ public void LoadSequences()
         StartCoroutine(ResolveSequenceFiles());
     }
 
+    /**
+     * <summary>
+     * Load and instantiate the Segments list
+     * </summary>
+     */
+    public void LoadSegments()
+    {
+        StartCoroutine(InstantiateSegments());
+    }
+
+    /**
+     * Load and instantiate the prefabs relevant to the currently displayed tab
+     * (used with the reload button)
+     */
+    public void LoadCurrentTab()
+    {
+        if (sequencesPanel.activeSelf)
+        {
+            LoadSequences();
+        }
+        else
+        {
+            LoadSegments();
+        }
+    }
+
     public void Awake()
     {
         DontDestroyOnLoad(gameObject);
     }
 
     public void Start()
     {
-        LoadSequences();
+        if (sequencesPanel == null)
+        {
+            Debug.LogError("RGSequenceManager is missing its sequencesPanel");
```

Comment:
again, these should be hard exceptions if this is null, our SDK is mis-configured/broken if this is happening